src/screens/Landing/index.js

```diff
@@ -105,6 +105,19 @@ export default function Landing() {
           vehicle: { vehicleType: VehicleTypeMap[vehicleTypeParam] ?? 'cuv' },
         },
       );
+    } else if (workflow === Workflows.CAPTURE_VEHICLE_SELECTION && !route.params?.captureComplete) {
+      if (vehicleTypeParam) {
```

Comment:
Show `CAPTURE_VEHICLE_SELECTION` screen in following scenarios
* If `v` param is not in the path and `vehicleTypeParam` value is undefined
* If `v` param is in the path, but `vehicleTypeParam` value is invalid

Reply:
Done.